Tests/SwiftFusionTests/Core/JacobianTests.swift

```diff
@@ -39,21 +40,16 @@ class JacobianTests: XCTestCase {
     }
 
     let j = jacobian(of: f, at: pts)
-    
-    let expected: [Array<Pose2>.TangentVector] = [
-      [Pose2.TangentVector(t_: Point2.TangentVector(x: 0, y: -1.0), rot_: 2.0), Pose2.TangentVector(t_: Point2.TangentVector(x: 0, y: 1.0), rot_: 0.0)],
-      [Pose2.TangentVector(t_: Point2.TangentVector(x: 1.0, y: 0), rot_: -2.0), Pose2.TangentVector(t_: Point2.TangentVector(x: -1.0, y: 0), rot_: 0.0)],
-      [Pose2.TangentVector(t_: Point2.TangentVector(x: 0.0, y: 0.0), rot_: -1.0), Pose2.TangentVector(t_: Point2.TangentVector(x: 0.0, y: 0.0), rot_: 1.0)]
-    ]
-    
-    // TODO(fan): Find a better way to do approximate comparison
-    XCTAssert(
-      expected.recursivelyAllKeyPaths(to:Double.self)
-        .map {j[keyPath: $0] - expected[keyPath: $0]}
-        .reduce(0.0, {$0 + abs($1)}) < 1e-10
-    )
-  }
-  
+
+    let expected = Tensor<Double>([
+      [0.0, -1.0, -2.0, 1.0, 0.0, 0.0],
```

Comment:
Should this stuff not be in Pose2Tests? Also, we should probably discuss the return type. This is a two argument function, so in GTSAM we would compute 2 3x3 Jacobians. I imagine we will refine/change this as we do a TDD development of Gauss-Newton optimization. For CGLS we don’t need this probably.

Reply:
A `jacobian` that deals with 2-argument functions by returning 2 jacobian matrices should be pretty easy to define! I'll file a ticket for this.

Reply:
Also I'll move these to `Pose2Tests`